### DISCUSSION TITLE
How to remove the active/open apps widget on the i3status or i3blocks panel?

### DISCUSSION THREAD

**Discussion by rendick** [i3/i3#5795] [Q&A]
my i3 config files: https://github.com/rendick/dotfiles/tree/main/i3

![image](https://github.com/i3/i3/assets/91757099/a198f7d6-b462-473c-9048-7f8fa2372b82)

**Comment by xianyuanlu** [comment]
tray_output none

**Reply by orestisfl** [reply]
🤦 yeah forgot there is a setting
